website/docs/api-backends.md

```diff
@@ -152,11 +152,12 @@ Given an `(M, N)` sequence of vectors, return an `(M, N*(nW*2+1))` sequence. The
 new sequence is constructed by concatenating `nW` preceding and succeeding
 vectors onto each column in the sequence, to extract a window of features.
 
-| Argument    | Type              | Description                                                       |
-| ----------- | ----------------- | ----------------------------------------------------------------- |
-| `seq`       | <tt>Floats2d</tt> | The original sequence .                                           |
-| `nW`        | <tt>int</tt>      | The window size.                                                  |
-| **RETURNS** | <tt>Floats2d</tt> | The created sequence containing preceding and succeeding vectors. |
+| Argument    | Type                      | Description                                                       |
+| ----------- | ------------------------- | ----------------------------------------------------------------- |
+| `seq`       | <tt>Floats2d</tt>         | The original sequence .                                           |
+| `nW`        | <tt>int</tt>              | The window size.                                                  |
+| `lengths`   | <tt>Optional[Ints1d]</tt> | Sequence lengths, introduces padding around sequences.            |
```

Comment:
`lengths` is a kwd-only arg, right? You can add a divider `_keyword-only_`

Reply:
Ah, that's nice! Fixed.